ir_beacon/models/ir_beacon/model.sdf

```diff
@@ -4,24 +4,25 @@
 <model name="ir_beacon">
   <static>true</static>
   <pose>0.0 0.0 0.0 0 0 0</pose>
-  <link name="link">
-    <pose>0.0 0.0 0.025 0 0 0</pose>
+  <link name="led_mount">
+    <pose>0.0 0.0 0.0 0 0 0</pose>
+    <!-- Disabling collision to put it under the drone w/o needing to change the drone for a box collision.
     <collision name="collision">
       <geometry>
         <box>
-          <size>0.1 0.1 0.05</size>
+          <size>0.1 0.1 0.005</size>
         </box>
       </geometry>
-    </collision>
+    </collision> -->
     <visual name="visual">
```

Comment:
not sure if it's intended but the bottom half of the beacon is now below the ground plane. You can raise it a little with these changes:

~~~
diff --git a/ir_beacon/models/ir_beacon/model.sdf b/ir_beacon/models/ir_beacon/model.sdf
index f5b8ca8..3bc3ec3 100644
--- a/ir_beacon/models/ir_beacon/model.sdf
+++ b/ir_beacon/models/ir_beacon/model.sdf
@@ -15,6 +15,7 @@
       </geometry>
     </collision> -->
     <visual name="visual">
+      <pose>0.0 0.0 0.0025 0 0 0</pose>
       <geometry>
         <box>
           <size>0.1 0.1 0.005</size>
~~~

Reply:
Thanks updated in d2437ac